tests/snakemake/conftest.py

```diff
@@ -41,7 +41,7 @@
 import pytest
 
 
-@pytest.fixture()
+@pytest.fixture
```

Comment:
Either `pytest.fixture` or `pytest.fixture()` work - I thought the convention was to have the parentheses present even if no arguments are passed, and we have that elsewhere. I'd prefer consistency if possible, so we should decide on one or other option.

Is there an argument for preferring one over the other, in general?

Reply:
Something in ruff apparently thinks without is better - I did not look into the rational, but they will be consistent.

Reply:
See 80fa7a908debc5c8605028c34ea00d30250afca2 from #66 which changed the rest

Reply:
So it did… as [ruff notes](https://docs.astral.sh/ruff/rules/pytest-fixture-incorrect-parentheses-style/), what matters is consistency and it's a style choice.